rest/src/main/java/fi/pyramus/rest/StudentRESTService.java

```diff
@@ -1172,6 +1174,15 @@ public Response listStudentGroupStudents(@PathParam("ID") Long id) {
     if (studentGroupStudents.isEmpty()) {
       return Response.noContent().build();
     }
+    
+    Collections.sort(studentGroupStudents, new Comparator<StudentGroupStudent>() {
```

Comment:
Why not do this in the database? It's much faster. Can be achieved in @OneToMany column using @OrderBy